src/libs/island/src/ISLAND.cpp

```diff
@@ -775,7 +775,7 @@ bool ISLAND::Mount(const std::string_view &fname, const std::string_view &fdir,
     if (pNode)
         seabed_id = pNode->Unlink2Model();
     else
-        core.Trace("Island: island %s has no sea bed, check me!", fname);
+        core.Trace("Island: island %s has no sea bed, check me!", std::string(fname).c_str());
```

Comment:
Why not `fname.data()`?

Reply:
because std::string_view has no guarantee that the data is null-terminated. https://eel.is/c++draft/string.view#access-14
actually, that's the key feature of string_view

Reply:
to the point, ideally we should switch to spdlog/fmt formatting completely; that would also solve such kind of issues. recent spdlog update targets fmt 8 support that I believe can help a lot with logging refactoring since it brings stable consteval formatting lexic parser